### PR TITLE
DBZ-6824 trim default value for the BIGINT and SMALLINT types when parsing MySQL DDL

### PR DESCRIPTION
**Pull request by ruanhang1993**
[Fix DBZ-6824.](https://issues.redhat.com/browse/DBZ-6824)

### CODE REVIEW

File: debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlDefaultValueConverter.java [hunk 1]
```diff
@@ -62,7 +62,7 @@ public class MySqlDefaultValueConverter implements DefaultValueConverter {
     @Immutable
     private static final Set<Integer> TRIM_DATA_TYPES = Collect.unmodifiableSet(Types.TINYINT, Types.INTEGER,
             Types.DATE, Types.TIMESTAMP, Types.TIMESTAMP_WITH_TIMEZONE, Types.TIME, Types.BOOLEAN, Types.BIT,
-            Types.NUMERIC, Types.DECIMAL, Types.FLOAT, Types.DOUBLE, Types.REAL);
+            Types.NUMERIC, Types.DECIMAL, Types.FLOAT, Types.DOUBLE, Types.REAL, Types.BIGINT, Types.SMALLINT);
```
Comment (jpechane) [inline debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlDefaultValueConverter.java:65]: Is now `TRIM_DATA_TYPES` a supreset of complete `NUMBER_DATA_TYPES`? If it is it probably makes sense to not repeat all the values but instead either make `TRIM_DATA_TYPES` of `NUMBER_DATA_TYPES` and the additional values or remove numeric types from this var and modify `convert` method to use both attributes.

**Comment by jpechane** [comment]
@ruanhang1993 Thanks for the PR, I left a recommendation for a small change reducing code duplication.

File: debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlDefaultValueConverter.java (hunk 1 above)
Comment (ruanhang1993) [reply debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlDefaultValueConverter.java]: Thanks for the suggestions. I have updated the PR and enriched the test.

**Review by Naros — APPROVED** [review]
Thanks, @ruanhang1993, the code changes look much better!

**Comment by Naros** [comment]
Applied to main (=2.4) and branch 2.3.

**Pull request merged**